backend/src/main/java/gov/cdc/usds/simplereport/config/WebConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package gov.cdc.usds.simplereport.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpStatus;
+import org.springframework.orm.jpa.support.OpenEntityManagerInViewFilter;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@ConditionalOnWebApplication
+public class WebConfiguration implements WebMvcConfigurer {
+
+    public static final String HEALTH_CHECK = "/health";
+
+    @Override
+    public void addViewControllers(ViewControllerRegistry registry) {
+        registry.addStatusController(HEALTH_CHECK, HttpStatus.OK);
```

Comment:
Does this hook into any Spring magic, or do component-level health checks need to be collected and invoked somewhere specific?

Reply:
This just adds a handler for the "/health" path that returns a 200 when the server is up. The actuator health check, which we are currently using as a liveness check, can be made more complicated by adding component checks to it, but then you have to decide if you want those on the internet or not. This does nothing but yes/no answers, which I think is in some ways better.